charts/jaeger/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: 1.53.0
+appVersion: 1.56.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
 type: application
-version: 3.0.0
+version: 3.0.1
```

Comment:
maybe avoid these changes with an otherwise administrative PR

Reply:
The chart version must be bumped. Hence makes sense to bump jaeger too.

Reply:
yes, but it should be a separate PR dedicated to bumping version

Reply:
@jkowall 
Newer Jaeger requires RBAC change as well. Please take a look https://github.com/jaegertracing/helm-charts/issues/549

Reply:
The chart version, indeed, needs to be bumped with every change. However, the Jaeger version bumped needs to be a separate PR.

Reply:
Reverted